ui/app/src/Views/SilenceList/SilenceView.elm

```diff
@@ -114,7 +117,7 @@ editButton silence =
         Expired ->
             a
                 [ class "btn btn-outline-info border-0"
-                , href (newSilenceFromMatchers silence.matchers)
+                , href recreateUrl
```

Comment:
Hey! What do you think about reusing the new silence page and extending it to support passing an optional comment field rather than introducing the whole new route?

Reply:
@w0rm Thank you for making a comment.
The reason why I introduce whole new route is to deal with possibilities change of SilenceView's field. This means to Avoid implement ``newSilenceFromMacherAndCommentAndXXXAndXXX..etc`` , when anyone need to add field in SilenceForm.

If low probability of extending new field in the future, implement ``newSilenceFromMatchersAndComment`` is simple and best way.
So, I can't make up my mind which is better.
Which do you think is better?

Reply:
Hey! Sorry for taking so long to respond. `newSilenceFromMatchersAndComment` sounds like a simpler and better option to me.

Reply:
Okey.
I’ll change this PR.
Wait a moment please.